### PR TITLE
Modify header so be like new-design

### PR DESCRIPTION
**Pull request by vandeput**

### CODE REVIEW

File: src/commons/components/Header/Header.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
 import React from 'react'
 import './Header.scss'
 import Logo from 'commons/assets/images/logo.png'
+import { Dropdown, Image, Menu, Segment } from 'semantic-ui-react'
+import faker from 'faker'
+
+const trigger = (
+  <span>
+    <Image avatar src={faker.internet.avatar()} />&nbsp;
+    <span>{faker.name.findName()}</span><br/>
```
Comment (nathanajah) [inline src/commons/components/Header/Header.js]: Lint - A space is required before the closing bracket (`<br />`)

File: src/commons/components/Header/Header.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
 import React from 'react'
 import './Header.scss'
 import Logo from 'commons/assets/images/logo.png'
+import { Dropdown, Image, Menu, Segment } from 'semantic-ui-react'
```
Comment (nathanajah) [inline src/commons/components/Header/Header.js]: Lint - Segment is defined but never used

**Comment by vandeput** [comment]
Fixed. By the way, which lint do we used? If I have time, I might want to setup the CI to also execute the lint.

**Comment by nathanajah** [comment]
We're using eslint. The command used can be found in package.json: 
`eslint bin build config server src tests`

**Comment by vandeput** [comment]
`$ npm run lint
> react-redux-starter-kit@3.0.0-alpha.2 lint /home/irvan/toki/judgels-web
> eslint bin build config server src tests`

**Pull request merged**